frontend/include/chpl/framework/mark-functions.h

```diff
@@ -158,6 +158,11 @@ template<typename ... Ts> struct mark<std::tuple<Ts...>> {
     mark_tuple_impl(context, keep, std::index_sequence_for<Ts...>());
   }
 };
+template<typename ... Ts> struct mark<const std::tuple<Ts...>> {
+  void operator()(Context* context, const std::tuple<Ts...>& keep) const {
+    mark_tuple_impl(context, keep, std::index_sequence_for<Ts...>());
+  }
+};
```

Comment:
This looks identical to what is declared just a few lines up. What am I missing?

Reply:
It's specialized for `const std::tuple`.

Reply:
Huh. I'm surprised you would need that.